internal/node/node.bzl

```diff
@@ -239,6 +239,7 @@ def _nodejs_binary_impl(ctx):
         "TEMPLATED_repository_args": _to_manifest_path(ctx, ctx.file._repository_args),
         "TEMPLATED_script_path": _to_execroot_path(ctx, ctx.file.entry_point),
         "TEMPLATED_vendored_node": "" if is_builtin else strip_external(ctx.file._node.path),
+        "TEMPLATED_pnp_file": _to_manifest_path(ctx, ctx.file.pnp_file),
```

Comment:
This doesn't need to be its own substitution: if we use this we don't use `TEMPLATED_bazel_require_script`, so you can replace that with this when `pnp_file` is set.

Reply:
I think we'd need to if we're going to keep this backwards compatible though

Reply:
I think I'd prefer if it has its own substitution, it makes it easier in the launcher script to verify if we are in PnP mode or not. WDYT?